python/nav/web/templates/webfront/_navbar_link_form.html

```diff
@@ -0,0 +1,17 @@
+{% load forms %}
+<div class="row">
+  {% for field in form %}
+    {% if field in form.attrs.form_fields %}
+      <div class="columns medium-5">
+        {% show_field field %}
+      </div>
+    {% elif field.name == 'DELETE' and navbar_formset.can_delete %}
+      <div class="columns link-delete medium-2">
+        <label>&nbsp;</label>
```

Comment:
SonarCloud complains about this label not being attached to any form control.  I know this was there in the original, but can you see any useful reason for it?